QUARKUS-6263.md

```diff
@@ -0,0 +1,61 @@
+# QUARKUS-6263 - Add OIDC Health Check
+
+JIRA: https://issues.redhat.com/browse/QUARKUS-6263
+
+Upstream issues: https://github.com/quarkusio/quarkus/issues/47861
+
+Upstream PR: https://github.com/quarkusio/quarkus/pull/47830
+
+## Scope of the testing
+
+- OIDC Health Check is correctly registered when `quarkus.oidc.health.enabled=true` and `quarkus-smallrye-health` dependency is present
+- OIDC Health Check is not registered if `quarkus.health.extensions.enabled=false`
```

Comment:
> OIDC Health Check is not registered if `quarkus.health.extensions.enabled=false`

I looked here https://quarkus.io/guides/all-config and there is no such a property.

Also, can you give me your thoughts on why you prefer to test that property over `quarkus.oidc.health.enabled=false`? I am open to this, but I need to understand first...